src/main/kotlin/week1/racingcar/controller/RaceController.kt

```diff
@@ -0,0 +1,24 @@
+package week1.racingcar.controller
+
+import week1.racingcar.domain.Car
+import week1.racingcar.domain.RacingGame
+import week1.racingcar.view.InputView
+import week1.racingcar.view.ResultView
+
+class RaceController {
+
+    fun startRacing() {
+        val raceInfo = InputView.getValidateRaceInfo()
+        val racingGame = RacingGame(raceInfo.cars.map { Car(it) }, raceInfo.timesToRace)
```

Comment:
List<Car>를 만드는 로직을 private 메서드로  뽑으면 조금 더 깔끔할 것 같아요. 👀
```suggestion
        val racingGame = RacingGame(createCars(raceInfo.cars), raceInfo.timesToRace)
```